github/VisualStudio
# Don't append name of repository to clone path

### What this PR does

We now specify the full repository path rather than the parent directory and a repository name. Previously it would always clone into a directory with name of the repository.

- Remove `repositoryName` parameter from `IRepositoryCloneService.CloneRepository`
- Clone to `repositoryPath` rather than `{repositoryPath}\{repositoryName}`

### How to test

1. Open `Clone a Repository dialog`
2. Select a repository to clone
3. Click `Clone` button
4. Check repository is cloned to the specified `Path`

Fixes #1913
